virt/live_migration/virt-live-migration.adoc

```diff
@@ -4,7 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-live-migration
 toc::[]
 
+.Prerequisites
+* Before using LiveMigration, ensure that the storage class used by the
+virtual machine has a persistent volume claim (PVC) with a shared
+ReadWriteMany (RWX) access mode.
+See xref:../../virt/virtual_machines/virtual_disks/virt-storage-defaults-for-datavolumes.adoc#virt-storage-defaults-for-datavolumes[Storage defaults for DataVolumes]
+to ensure all your storage settings are correct.
```

Comment:
Suggest:

s/all//

"to ensure your storage settings are correct."

Reply:
Done